src/lib/utils/translations.ts

```diff
@@ -12,12 +12,17 @@ export const languages: Languages = i18nConfig.reduce((result, config) => {
 export const isLangRightToLeft = (lang: Lang): boolean => {
   const langConfig = i18nConfig.filter((language) => language.code === lang)
 
-  if (!langConfig.length)
-    throw new Error("Language code not found in isLangRightToLeft")
+  if (!langConfig.length) {
+    return false
+  }
 
   return langConfig[0].langDir === "rtl"
 }
 
+export const filterFakedLocales = (locales: string[] | undefined) => {
```

Comment:
This feels like the output would be a list of faked locales from the name... we're filtering _for_ the real locales here so maybe `filterRealLocales`, `filterForRealLocales` or `filterOutFakedLocales`?